Sources/Core/Shared/GroupOperation.swift

```diff
@@ -185,22 +248,38 @@ public class GroupOperation: Operation, OperationQueueDelegate {
 
     /**
      The group operation acts as its own queue's delegate. When an operation is added to the queue,
-     assuming that the finishing operation has not started (or finished), and the operation is not
-     the finishing operation itself, then we add the operation as a dependency to the finishing
-     operation.
+     assuming that the finishing operation has not started (or finished), the operation is not
+     the finishing operation itself, and the CanFinishOperation has not yet started,
+     then we add the operation as a dependency to the CanFinishOperation.
```

Comment:
This is an odd issue regarding the docs. `CanFinishOperation` is a private class which the framework consumer has no idea about. Is there a way to describe the behaviour without mentioning this class?